src/types/capp.go

```diff
@@ -64,7 +64,7 @@ type CappUri struct {
 	CappName      string `uri:"cappName" binding:"required"`
 }
 
-type CappError struct {
+type MessageResponse struct {
```

Comment:
Seeing as this is more general than the rest of the types here. It should be moved to common.go